server/flaskr/database/postgres/handlers/user_data_handler.py

```
@@ -9,24 +11,47 @@ def create_user(cls, firstName: str, lastName: str, email: str, passwordHash: st
-
                 return User(userId, *params)
         except Exception as e:
             # TODO: Logging
             pass
         return None
 
     @classmethod
-    def get_users(cls):
-        query = f"SELECT * from users;"
-        users = read_query(query)
+    def add_user_to_org(cls, userId, orgId):
```

Ruff (0.15.15)

[warning] Missing return type annotation for classmethod `add_user_to_org`

Add return type annotation: `bool`

(ANN206)